src/local_scheduler/local_scheduler_extension.cc

```diff
@@ -89,13 +89,14 @@ static PyObject *PyLocalSchedulerClient_log_event(PyObject *self,
   int key_length;
   const char *value;
   int value_length;
-  if (!PyArg_ParseTuple(args, "s#s#", &key, &key_length, &value,
-                        &value_length)) {
+  double time;
+  if (!PyArg_ParseTuple(args, "s#s#d#", &key, &key_length, &value,
+                        &value_length, time)) {
```

Comment:
Check out the documentation for `PyArg_ParseTuple` on this page https://docs.python.org/3/c-api/arg.html.

It should probably be `s#s#d`, and then pass in `&time` (because we need the value to be written to the `time` variable, so we need to pass in its address).